endpoints/openrtb2/auction.go

```diff
@@ -290,6 +290,10 @@ func (deps *endpointDeps) validateRequest(req *openrtb.BidRequest) []error {
 		if err := validateBidAdjustmentFactors(bidExt.Prebid.BidAdjustmentFactors, aliases); err != nil {
 			return []error{err}
 		}
+
+		if err := validateSChains(bidExt); err != nil {
+			return []error{err}
+		}
```

Comment:
`*openrtb_ext.ExtRequest` also gets unmarshaled and validated in files `endpoints/openrtb2/amp_auction.go` and `endpoints/openrtb2/video_auction.go` we could probably also `validateSChains(bidExt)` from there in order to prevent a defective "ext"JSON field from getting all the way to `splitBidRequests`

Reply:
The `openrtb2/amp_auction.go` and `openrtb2/video_auction.go` endpoints both call the `validateRequest` method in `openrtb2/auction.go` which calls `validateSChains` so we're covered. Those endpoints forgo processing if the method returns an endpoint as well.

Reply:
Ohh I see